docs/resources/task.md

```diff
@@ -15,7 +15,6 @@ resource "iterative_task" "task" {
 
   environment = {GREETING = "Hello, world!"}
   directory   = "${path.root}/shared"
-
```

Comment:
> * _When multiple arguments with single-line values appear on consecutive lines at the same nesting level, align their equals signs._
>
> * _When both arguments and blocks appear together inside a block body, place all of the arguments together at the top and then place nested blocks below them. Use one blank line to separate the arguments from the blocks._
> 
> * _Use empty lines to separate logical groups of arguments within a block._
>
> [**(Terraform Style Conventions)**](https://www.terraform.io/docs/language/syntax/style.html)

I would keep the blank line, but, if you prefer not to, equal signs should be aligned. If you don't believe me, ask [`terraform fmt`](https://www.terraform.io/docs/cli/commands/fmt.html) for a second opinion. 😉

Reply:
in that case all the snippets need updating... 9a25d0c

Reply:
🙈 A good prize for being pedantic. 😄 I recall fixing some of them manually when reviewing #237 though.